### PR TITLE
[WIP][ADD] manual own checks + third checks

### PR DESCRIPTION
**Pull request by jjscarafia**
Description of the issue/feature this PR addresses:

Current behavior before PR:

Desired behavior after PR is merged:

--
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

File: addons/account/models/account_payment.py [hunk 1]
```diff
@@ -819,6 +823,7 @@ def _create_paired_internal_transfer_payment(self):
                 'payment_type': payment.payment_type == 'outbound' and 'inbound' or 'outbound',
                 'move_id': None,
                 'ref': payment.ref,
+                'date': payment.date,
```
Comment (jco-odoo) [inline addons/account/models/account_payment.py:826]: @smetl Can you check this if we can do this as a generic fix?

Comment (smetl) [reply addons/account/models/account_payment.py:826]: Idk if this is intended or not :/

Comment (jjscarafia) [reply addons/account/models/account_payment.py:826]: without this, if you create a transfer for a date different from today, the paired transfer will be created for today. IMHO, that would not be the expected behavior, with this change, both payments (of type transfer) are created for on the same date

**Comment by jjscarafia** [comment]
closing it!

**Pull request closed without merging**